Wrote initial draft of 2.8: logging

Duckapple opened the pull request.

Duckapple committed:
Wrote initial draft of 2.8: logging

Duckapple committed:
## Process Logging -> ## Logging

Jakobis commented on report/28_Process-Logging.md:
```diff
@@ -1 +1,23 @@
-What do you log in your systems and how do you aggregate logs?
+## Process Logging
+
+Logging output from processes is important in order to know what is going on in the system. If your program in no way outputs any information, then it is hard to diagnose faults or detect attacks, which can make fixing those things harder. 
```
"a program" i stedet for "your program"

Jakobis commented on report/28_Process-Logging.md:
```diff
@@ -1 +1,23 @@
+
+Logs alone can be daunting to search through and store. Search engines for looking through logs are viable for being able to extract information like quantity, similarity and frequency of events, and for this, we use [Elasticsearch](https://aws.amazon.com/elasticsearch-service/the-elk-stack/what-is-elasticsearch/) with [Kibana](https://www.elastic.co/kibana). Elasticsearch is one of the leading o ...
+
+We did not actually set up any visualizations or use the logs for anything, however. We made it work for illustration purposes, but in part due to unfamiliarity with the Kibana interface, there were never any visualizations set up of the data that had been logged. When writing this report, some visualizations were made and saved, and finding features in the UI such as the save button location, se ...
```
Illustration purposes?

Duckapple committed:
Made sentences easier to comment on

This was done by putting each sentence on different lines, which does not change the result.

Duckapple committed:
Corrections based on Jakob's feedback

Duckapple commented:
closes #216

Duckapple committed:
Changed heading level to h3

Hjaltesorgenfrei commented on report/28_Process-Logging.md:
```diff
@@ -1 +1,38 @@
+- Getting messages for follows, including count (again).
+- Any of the calls made when unauthorized.
+
+This logging is done all with the information level, and it makes sense: 
```
Does it make sense? Also Information level needs some context.

tobloef replied:
Yeah, der er nok noget der skal omskrives her, det der kolon er også lidt weird.

Hjaltesorgenfrei committed:
remove some text
